src/main.rs

```diff
@@ -99,6 +99,18 @@ pub struct Args {
     )]
     pub json_expand_depth: Option<usize>,
 
+    #[arg(
+        short = 's',
```

Comment:
`-s` will be used at https://github.com/ynqa/jnv/issues/29.

Reply:
@edy555 ...No problem. I've decided not to worry about it until we actually encounter conflicts. Please note that there may be changes in the names of the short flags in the future.